draft-kazuho-httpbis-priority.md

```diff
@@ -184,21 +244,21 @@ For example, when the client sends an HTTP request with
 :scheme = https
 :authority = example.net
 :path = /menu.png
-priority = urgency=non-blocking, progressive=?1
+priority = urgency=3, progressive=?1
 ~~~
 
 and the origin responds with
 
 ~~~ example
 :status = 200
 content-type = image/png
-priority = urgency=document
+priority = urgency=1
 ~~~
 
-the intermediary's understanding of the urgency is promoted from `non-blocking`
-to `document`, because the server-provided value overrides that provided by the
-client.  The progressiveness continues to be `1`, the value specified by the
-client, as the server did not specify the `progressive` parameter.
+the intermediary's understanding of the urgency is promoted from `3` to `1`,
+because the server-provided value overrides that provided by the client.  The
```

Comment:
> overrides that
-> overrides the one

Reply:
Would "one" be confusing with "`1`" being used right before?

Maybe it would be best to simply write "because the server-provided value overrides the client-provided value".

Reply:
ah, I can see how that's confusing :) 

I meant "the one" as referring to the value as a concept, not the value 1. 
As in: the server-provided value overrides the value provided by the client, and not: the server-provided value overrides the 1 provided by the client. 

So maybe just make it "overrides the value" then

Reply:
Makes sense. Thank you for the suggestion. Applied in d7135215.